src/utils/workers.ts

```diff
@@ -37,7 +38,20 @@ export const computeWorkersData = (data: MiningData[]) => {
   const bestever = data.reduce((acc, worker) => Math.max(acc, worker.bestever), 0);
   const lastshare = Math.max(...data.map((worker) => worker.lastshare));
   const authorised = data.reduce((acc, worker) => acc + worker.authorised, 0);
-  const flattenedWorkers = data.map((miningData) => miningData.worker).flat();
+
+  const flattenedWorkers = data.map((miningData) =>
+    miningData.worker.map((wrkr) => {
+
+      const matchingWorker = ENV.WORKERS.find(
+        (envWorker: { name: string,  address: string; }) => envWorker.address === wrkr.workername.replace(".bitaxe", "")
+      );
+
+      wrkr.username = matchingWorker ? matchingWorker.name : wrkr.workername;
+      wrkr.offline = parseInt(wrkr.hashrate1m) == 0;
```

Comment:
If you don't care about type check (you are doing == instead of ===) then you don't need parseInt